<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SixDegreesofSeperationClass</class>
 <widget class="QMainWindow" name="SixDegreesofSeperationClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>619</width>
    <height>459</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SixDegreesofSeperation</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="SeperationMenuButton">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>70</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Degrees of Seperation</string>
    </property>
   </widget>
   <widget class="QPushButton" name="MoviesMenuButton">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>140</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Movies</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ActorsMenuButton">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>210</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Actors</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ExitMenuButton">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>280</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QLabel" name="titleLabel">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>10</y>
      <width>481</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:26pt; font-weight:600; text-decoration: underline;&quot;&gt;Six Degrees of Seperation&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="informationLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>221</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;University: Nottingham Trent University&lt;/p&gt;&lt;p&gt;Student ID: N0633796&lt;/p&gt;&lt;p&gt;Student Name: Bartlomiej Rembisz&lt;/p&gt;&lt;p&gt;Module: Software Design and Implementation&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="fileStatus">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>380</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Not loaded</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>380</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>File status:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>619</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="SixDegreesofSeperation.qrc"/>
 </resources>
 <connections/>
</ui>
